README.md

```diff
@@ -1,70 +1,141 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# 📗 Table of Contents
```

Comment:
- [ ] OPTIONAL: The doctors sometimes don't show up in the homepage:

![image](https://github.com/tajemouti/medical-appointments-front-end/assets/57421823/7df71711-42ca-4df5-ba13-117dc8ea105c)

Kindly, I suggest take a look into this.

I'm marking this as optional because it worked for me at first, but after inspending some time with the app, it stopped showing the doctors on the homepage.

In the database, there are 5 doctors:

![image](https://github.com/tajemouti/medical-appointments-front-end/assets/57421823/eed86a9a-bf56-4611-a42d-9490be31434c)

Apart from the above issue, only 3 of the 5 doctors are showing up reservations, and in "delete doctor":

![image](https://github.com/tajemouti/medical-appointments-front-end/assets/57421823/7d234a50-83d0-4b58-9820-0eb6791de093)

![image](https://github.com/tajemouti/medical-appointments-front-end/assets/57421823/165a0403-4b05-47f6-be30-7b6380b7cb7b)

I highly suggest you take a look at this, in order to ensure success in the final assessment. 👍

Reply:
Hello @Meltrust,
Thank you so much for taking the time to review our final capstone project, and also for your insightful and cheerful feedback.
And regarding the issues you mentioned above, we think that it is a network issue, otherwise the application is working fine locally.
We have recorded this [Loom](https://www.loom.com/share/a9e4c5bdab2a4ac38a3038c87cc1743c?sid=3843cbeb-01e4-4858-9054-0a5e91bf7350) in which application is working without a problem. Please check it out.

Have a great time ahead,
Cheers